File: Human.cpp
/*Saya Raffi Ardhi Naufal NIM 2202495 mengerjakan Latihan Praktikum 1 dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.*/
#include <iostream>
#include <string>

#pragma once

using namespace std;

class Human
{
    private:
        string id;
        string name;
        string namBid;
        string namPar;

    public:
        Human()
        {
            this->id = "";
            this->name = "";
            this->namBid = "";
            this->namPar = "";
        }

        Human(string id, string name, string namBid, string namPar)
        {
            this->id = id;
            this->name = name;
            this->namBid = namBid;
            this->namPar = namPar;
        }

        string get_id()
        {
            return this->id;
        }

        void set_id(string id)
        {
            this->id = id;
        }

        string get_name()
        {
            return this->name;
        }

        void set_name(string name)
        {
            this->name = name;
        }

        string get_namBid()
        {
            return this->namBid;
        }

        void set_namBid(string namBid)
        {
            this->namBid = namBid;
        }

        string get_namPar()
        {
            return this->namPar;
        }

        void set_namPar(string namPar)
        {
            this->namPar = namPar;
        }

        // void add()

        // void eat()
        // {
        //     cout << this->name << " is eating!" << '\n';
        // }

        // void sleep() 
        // {
        //     cout << this->name << " is sleeping!" << '\n';
        // }

        // void tambah(list<Human>& llist)
        // {
        //     string id,name,namBid,namPar;
        //     cout << "Masukkan id, nama, nama bidang dan nama partai dari anggota yang ingin dimasukkan : " << '\n';
        //     cin >> id >> name >> namBid >> namPar;

        //     llist.push_back(Human(id, name, namBid, namPar));
        // }

        ~Human()
        {

        }
};

File: PHP/Program/Human.php
<!-- // Saya Raffi Ardhi Naufal NIM 2202495 mengerjakan Latihan Praktikum 1 dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.  -->
<?php

// Definisi kelas Human
class Human {
    // Properti pribadi untuk menyimpan data anggota DPR
    private $id;
    private $name;
    private $namBid;
    private $namPar;
    private $profilePhoto;

    // Konstruktor untuk inisialisasi objek Human dengan data awal atau nilai default
    public function __construct($id = "", $name = "", $namBid = "", $namPar = "", $profilePhoto = "") {
        $this->id = $id;
        $this->name = $name;
        $this->namBid = $namBid;
        $this->namPar = $namPar;
        $this->profilePhoto = $profilePhoto;
    }

    // Metode-metode getter dan setter untuk mengakses dan mengubah nilai properti
    public function getId() {
        return $this->id;
    }

    public function setId($id) {
        $this->id = $id;
    }

    public function getName() {
        return $this->name;
    }

    public function setName($name) {
        $this->name = $name;
    }

    public function getNamBid() {
        return $this->namBid;
    }

    public function setNamBid($namBid) {
        $this->namBid = $namBid;
    }

    public function getNamPar() {
        return $this->namPar;
    }

    public function setNamPar($namPar) {
        $this->namPar = $namPar;
    }

    public function getProfilePhoto() {
        return $this->profilePhoto;
    }

    public function setProfilePhoto($profilePhoto) {
        $this->profilePhoto = $profilePhoto;
    }
}

?>